tests/test_core.py

```diff
@@ -1700,21 +1700,6 @@ def test_memorygrowth(self):
     # Win with it
     self.emcc_args += ['-s', 'ALLOW_MEMORY_GROWTH=1']
     self.do_run(src, '*pre: hello,4.955*\n*hello,4.955*\n*hello,4.955*')
-    win = open('src.cpp.o.js').read()
```

Comment:
this test code relied on names not being minified, and is not important anyhow - it tests an aspect of asm.js memory growth, which is deprecated